src/drivers/pci.rs

```diff
@@ -519,6 +537,7 @@ pub(crate) fn get_interrupt_handlers() -> HashMap<InterruptLine, InterruptHandle
 
 #[cfg(all(
 	not(all(target_arch = "x86_64", feature = "rtl8139")),
+	feature = "virtio-net",
 	any(feature = "tcp", feature = "udp")
 ))]
```

Comment:
Do we still need `any(feature = "tcp", feature = "udp")` here and at the other places?